components/tooltip/tooltip-help.js

```diff
@@ -0,0 +1,83 @@
+import '../colors/colors.js';
+import '../tooltip/tooltip.js';
+import { css, html, LitElement } from 'lit';
+import { bodySmallStyles } from '../typography/styles.js';
```

Comment:
I think this can be removed now since it's no longer being used.  Can also be removed from the static styles assignment.